projects/Mallard/src/components/lists/list.tsx

```diff
@@ -26,43 +27,54 @@ export const ListHeading = ({ children }: { children: string }) => (
         </SafeAreaView>
     </View>
 )
-export class List extends React.Component<PropTypes> {
-    render() {
-        const { data, onPress } = this.props
-        const Highlight =
-            Platform.OS === 'android'
-                ? TouchableNativeFeedback
-                : TouchableHighlight
-        return (
-            <FlatList
+
+const ListItem = ({
+    onPress,
+    item: { title, explainer, data },
+}: {
+    item: Item
+    onPress: OnPressHandler
+}) => {
+    const Highlight =
+        Platform.OS === 'android' ? TouchableNativeFeedback : TouchableHighlight
+    return (
+        <Highlight onPress={() => onPress(data)}>
+            <View
                 style={{
-                    borderTopWidth: 1,
-                    borderColor: '#ddd',
+                    padding: metrics.horizontal,
+                    paddingVertical: metrics.vertical,
+                    borderTopWidth: 0,
+                    borderBottomWidth: StyleSheet.hairlineWidth,
+                    borderColor: useAppAppearance().borderColor,
+                    backgroundColor: useAppAppearance().backgroundColor,
```

Comment:
Would there be any benefit in caching the return value of `useAppAppearance()`?

Reply:
yes! also in using stylesheet.create outside of this, like the article page